api/src/main/java/com/basistech/rosette/api/RosetteAPI.java

```diff
@@ -1216,4 +1793,11 @@ private String headerValueOrNull(Header header) {
     public void close() throws IOException {
         httpClient.close();
     }
+
+    private DocumentRequest.BaseBuilder getDocumentRequestBuilder() throws IOException {
+        return new DocumentRequest.Builder()
+                .language(language)
+                .genre(genre)
+                .options(options);
```

Comment:
I apologize for the trickling comments. Can you update this with null checks and if null don't add to the builder? I'm not 100% sure at the moment how the server does differently if the json attribute is not set vs set to null.